Update templates (#660)
tauri-apps/tao: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve Tauri

Reviewer: ```suggestion
about: Create a report to help us improve tao
```

title: ''

> labels: bug

Reviewer: ```suggestion
labels: 'type: bug'
```

assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

**Steps To Reproduce**

> Steps to reproduce the behavior. It **must** uses tao directly instead of just tauri dependency.

Reviewer 2: ```suggestion
Steps to reproduce the behavior. It **must** use tao directly instead of tauri.
```
